docs/product-development-guide/src/main/asciidoc/kie-server-clients-ref.adoc

```diff
@@ -0,0 +1,23 @@
+
+* https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.5.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/JobServicesClient.java[JobServicesClient]: Used to schedule, cancel, re-queue, and get job requests.
+* https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.5.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/ProcessServicesClient.java[ProcessServicesClient]: Used to start, signal, and abort processes or work items.
+* https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.5.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/QueryServicesClient.java[QueryServicesClient]: Used to query processes, process nodes, and process variables.
+* https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.5.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/RuleServicesClient.java[RuleServicesClient]: Used to send commands to the server to perform rule-related operations (for example, insert objects into the working memory or execute rules).
+* https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.5.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UserTaskServicesClient.java[UserTaskServicesClient]: Used to perform all user-task operations (such as to start, claim, or cancel a task) and query tasks by specified field (such as process instances ID or user).
+* https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.5.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UIServicesClient.java[UIServicesClient]: Used to get String representation of forms (XML or JSON) and of the process image (SVG).
+* https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.5.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/SolverServicesClient.java[SolverServicesClient]: Used to perform all {PLANNER} operations, such as getting the solver state and the best solution, or disposing of a solver.
+
+* https://github.com/kiegroup/droolsjbpm-integration/blob/7.3.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/admin/ProcessAdminServicesClient.java[ProcessAdminServicesClient]: Provides an interface for operations with process instances.
+
+* https://github.com/kiegroup/droolsjbpm-integration/blob/7.3.x/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/admin/UserTaskAdminServicesClient.java[UserTaskAdminServicesClient]: Provides an interface for operations with user tasks.
```

Comment:
For continuity, shouldn't the description be like - "Used to provide an interface for operations with user tasks."
Alternate suggestion is the same as above one.